packit_service/worker/build/build_helper.py

```diff
@@ -91,6 +96,43 @@ def local_project(self) -> LocalProject:
             )
         return self._local_project
 
+    @property
+    def pr_id(self) -> int:
+        if self._pr_id is None:
+            try:
+                self._pr_id = int(self.metadata.event_dict["identifier"])
```

Comment:
@lbarcziova do we have `pr_id` in the `event_dict`? (So we don't need to do this `try-except`.) Thanks

Reply:
it depends on the content of the original Event, but in EventData (self.metadata) we are trying to get both [pr_id](https://github.com/packit-service/packit-service/blob/398d0d20060ea3af92d21d9dab9b3adef599f365/packit_service/service/events.py#L186)  and [identifier](https://github.com/packit-service/packit-service/blob/398d0d20060ea3af92d21d9dab9b3adef599f365/packit_service/service/events.py#L188), so this could be definitely simplified

Reply:
@lbarcziova Thanks.